mcs/tools/mono-api-html/MarkdownFormatter.cs

```diff
@@ -191,7 +191,7 @@ public override void DiffRemoval (StringBuilder output, string text, bool breaki
 
 		public override void Diff (TextWriter output, ApiChange apichange)
 		{
-			foreach (var line in apichange.Member.ToString ().Split ('\n')) {
+			foreach (var line in apichange.Member.ToString ().Split (new[] { "\r\n", "\n" }, 0))) {
```

Comment:
Here I split on both, but technically I could split on `Environment.NewLine` since the way this property is created is with a `StringBuilder` and the `AppendLine` method.